openapi.yml

```diff
@@ -0,0 +1,381 @@
+openapi: '3.0.2'
```

Comment:
> String does not match the pattern of "^3\.1\.\d+(-.+)?$".yaml-schema: openapi.json
Source: [schema.json](https://raw.githubusercontent.com/OAI/OpenAPI-Specification/main/schemas/v3.1/schema.json)

Reply:
OpenAPI 3.0.2 was still a valid version, I've updated to 3.1.0 now though